backend/routers/auth.router.js

```diff
@@ -12,18 +13,18 @@ router.use(function (req, res, next) {
 
 router.post(
   '/signup',
-  [userController.validateCreateUserAPICall, verifyUser.checkDuplicateEmail],
-  userController.createUser,
+  [authAPIValidator.validateCreateUserAPICall, verifyUser.checkDuplicateEmail],
+  UserController.createUser,
 );
 
 router.post(
-  "/signin",
-  [userController.validateSigninUserAPICall, verifyUser.isAdminByEmail],
-  userController.signin
+  '/signin',
+  [authAPIValidator.validateSigninUserAPICall, verifyUser.isAdminByEmail],
+  UserController.signin,
 );
 
-router.post("/verify-signin", userController.verifySignIn);
+router.post('/verify-signin', UserController.verifySignIn);
 
-router.post('/me', [authJwt.verifyCookie], userController.verifyMe);
+router.post('/me', [verifyAuth.verifyCookie], UserController.verifyMe);
```

Comment:
From my side, `verifyAuth` is not a very appropriate name. It starts from a verb and sounds like a method name. What about `AuthVerifier` or `AuthUtil`? Or chose what you like in a noun style.

Reply:
👍